manifester/settings.py

```diff
@@ -2,7 +2,7 @@
 import os
 from pathlib import Path
 
-from dynaconf import Dynaconf, Validator
+from dynaconf import LazySettings, Validator
```

Comment:
Why this change?

Reply:
This change was part of my initial debugging. In trying to determine why my Vault integration was failing, I tried matching the Robottelo implementation as closely as possible. I can revert this.

Reply:
Yeah they're equivalent statements since `Dynaconf` is an alias to `LazySettings`. However, it's recommended to use `Dynaconf` here instead. Robottelo is likely just holding on to an old pattern (and should be changed).

https://github.com/dynaconf/dynaconf/blob/master/dynaconf/__init__.py#L28